sysfs/sysfssmoketest/benchmark.go

```diff
@@ -1,69 +0,0 @@
-// Copyright 2017 The Periph Authors. All rights reserved.
```

Comment:
sysfs/sysfssmoketest/benchmark.go is a remnant where I failed (forgot!) to move it to cmd. I'm sorry about that. I'd prefer to do this as a separate PR since this PR is already fairly big.

Reply:
restored.